lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -459,6 +460,37 @@ def _AnalyzeSerializedRuntime(serialized_runtime):
   return (kvm_cmd, kvm_nics, hvparams, kvm_disks)
 
 
+def _GetCacheSettings(cache_type):
+  """Return cache settings suitable for use with -blockdev
+
+  @param cache_type: string
+  @return: (writeback, direct, no_flush)
+  @rtype: tuple
+  """
+
+  if cache_type == constants.HT_CACHE_NONE:
+    return True, True, False
+  elif cache_type in [constants.HT_CACHE_WBACK, constants.HT_CACHE_DEFAULT]:
```

Comment:
Probably `gnt-instance(8)` should be adopted, too. ATM it's documented `default` means "to not pass any cache option". Here it's explicitly `writeback` (which might be the same as not passing any cache option?). Probably we also should think of better defaults. i.e. default to `none`?

Reply:
I have not yet done any documentation work (e.g. `io_uring` is also still missing). I will add that in one of the next iterations and tick the relevant checkbox of the top post.

Regarding `default` I think we should make the documentation more explicit. "not passing any cache option" also lead to explicit cache settings in QEMU before, they just depended on whatever default the QEMU version in question assumed (although I think not much has changed in that regard in the past years).